### PR TITLE
News page functionality

### PR DESCRIPTION
**Pull request by Karyum**
rel #104

### CODE REVIEW

File: src/routes/news.js [hunk 1]
```diff
@@ -0,0 +1,13 @@
+const { mockNews } = require('../../database/news_schema.js');
+
+module.exports = (req, res) => {
+  const id = req.params.id;
+  mockNews.findOne({_id:id}, (error, news) => {
+    if (error) res.render('not_found')
+    else {
+      res.render('news.hbs', {
```
Comment (Suuuuha) [inline src/routes/news.js]: why news.hbs?

Comment (Karyum) [inline src/routes/news.js]: @Suuuuha changed it

**Comment by esraajb** [comment]
@Karyum @Suuuuha  conflict

**Comment by Karyum** [comment]
@esraajb this one is sorted sorry about the others

File: src/actions/get_news.js [hunk 2]
```diff
@@ -0,0 +1,14 @@
+const { mockNews } = require('../../database/news_schema.js');
+
+const findNews = (id, hbs, res) => {
```
Comment (esraajb) [inline src/actions/get_news.js]: getNews

**Pull request merged**